api-platform/config/packages/doctrine.yaml

```diff
@@ -13,7 +13,7 @@ doctrine:
       App:
         is_bundle: false
         dir: '%kernel.project_dir%/src'
-        prefix: 'App\ApiResource'
+        prefix: 'App\Entity'
```

Comment:
check that the playground still work and rename guides namespace if needed (maybe it's done?)